proposals/NNNN-swiftpm-plugin-api-amendments.md

```diff
@@ -0,0 +1,481 @@
+# Amend SE-0303 Package Plugin API to Use `@main`
```

Comment:
is this file still needed?

Reply:
No, it will be removed.  The original pitch still references it so I will want to change the reference so that the link doesn't break, and then remove it.